src/Model/BrokenExternalLink.php

```diff
@@ -58,7 +58,7 @@ public function canEdit($member = false)
     public function canView($member = false)
     {
         $member = $member ? $member : Security::getCurrentUser();
-        $codes = array('content-authors', 'administrators');
+        $codes = array('content-authors', 'administrators', 'CMS_ACCESS_CMSMain');
```

Comment:
We're trying to move away  from the old array syntax. `'content-authors', 'administrators'` are not valid permissions. When you do a permission check on an admin user, it will always return true, no matter what permission your are checking, so you only need to check for one permission.

```suggestion
        $codes = ['CMS_ACCESS_CMSMain'];
```